test/utils/consts.go

```diff
@@ -37,4 +39,6 @@ const (
 var (
 	PtpGrandMasterPolicyName = "test-grandmaster"
 	PtpSlavePolicyName       = "test-slave"
+	PtpBcMasterPolicyName    = "test-bc-master"
```

Comment:
Couldn't we use old node labels for this ?

Reply:
I'd prefer to use a clearer label for for the specific role. Makes it easier to understand and configure